Adamant/Modules/Chat/ViewModel/ChatViewModel.swift

```diff
@@ -610,8 +722,258 @@ final class ChatViewModel: NSObject {
             return false
         }
         
+        guard nodesStorage.haveActiveNode(in: .adm) else {
+            dialog.send(.alert(ApiServiceError.noEndpointsAvailable(
+                coin: NodeGroup.adm.name
+            ).localizedDescription))
```

Comment:
Don't we want to move this part to the API layer? And it will return an error if there are no nodes

Reply:
we'll discuss